sql/pg_scripts/ght-restore-pg

```diff
@@ -55,30 +55,52 @@ fi
 # Convert to full path
 dumpDir="`pwd`/$dumpDir"
 
-if [ ! -e $dumpDir/schema.sql ]; then
-  echo "Cannot find $dumpDir/schema.sql to create DB schema"
+pgDir=`echo $dumpDir/pg_scipts`
```

Comment:
Why do we need a special directory for postgres scripts there? Isn't the default one already good?